java/src/jmri/server/json/time/JsonTimeHttpService.java

```diff
@@ -33,8 +33,8 @@ public JsonTimeHttpService(ObjectMapper mapper) {
     }
 
     @Override
-    // using @Nullable to override @Nonnull in super class
```

Comment:
This must be Nullable per https://static.javadoc.io/com.google.code.findbugs/jsr305/3.0.2/javax/annotation/Nullable.html and http://findbugs.sourceforge.net/manual/annotations.html since this is overriding a Nonnull annotation, and is called out for use in that manner by those documents.

Reply:
The linked docs are also consistent with using @CheckForNull here: "Use CheckForNull to indicate that the element value should always be checked for a null value."